management/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure Service Management packages for Go
+
+The *github.com/Azure/azure-sdk-for-go/management* packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](https://github.com/Azure/azure-sdk-for-go/tree/master/arm) folder.
+
+## First a Sidenote: Authentication and the Azure Service Manager
+
+The client currently supports authentication to the Service Management
+API with certificates or Azure `.publishSettings` file. You can 
+download the `.publishSettings` file for your subscriptions
+[here](https://manage.windowsazure.com/publishsettings).
+
+### Example: Creating a Linux Virtual Machine
+
+Complete source code for this example can be found in [createVM.go](https://github.com/Azure/azure-sdk-for-go/tree/master/management/examples/createVM/createVM.go).To try this example, [download your .publishSettings](https://manage.windowsazure.com/publishsettings) and add its path in [createVM_test.go](https://github.com/Azure/azure-sdk-for-go/tree/master/management/examples/createVM/createVM_test.go). Run the example with command `go test github.com/Azure/azure-sdk-for-go/management/examples/createVM -v`
```

Comment:
relative links note again.

also my comment about _test files... this shouldn't be a `go test` command